pkg/storaged/filesystem/utils.jsx

```diff
@@ -147,24 +147,35 @@ export const MountPoint = ({ fstab_config, forced_options, backing_block, conten
     }
 
     let extra_text = null;
-    if (!is_filesystem_mounted) {
+    if (client.in_anaconda_mode()) {
         if (!old_dir)
-            extra_text = _("The filesystem has no permanent mount point.");
-        else
-            extra_text = _("The filesystem is not mounted.");
-    } else if (backing_block != content_block) {
-        if (!opt_never_auto)
-            extra_text = _("The filesystem will be unlocked and mounted on the next boot. This might require inputting a passphrase.");
+            extra_text = _("The filesystem has no assigned mount point.");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19807-20240115-125110-9acae421-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)